### ISSUE TITLE
Assignment header

### ISSUE THREAD

**Issue by otytlandsvik** [td-org-uit-no/assignment-template-latex#2]
Add a header including course, assignment name and the current date, like on the [typst template](https://github.com/td-org-uit-no/assignment-template-typst).

**Comment by nohr-ai** [comment]
Should already be set.
I'm working with Mike on some significant changes that will clean up the template, makefile, and reproducible builds. I will double-check this and include a fix if it's missing.

**Comment by nohr-ai** [comment]
Right, the current header is for ieee `journal` style. I'll make sure to include working config for the `conference` style we are using now.

**Comment by nohr-ai** [comment]
To include running headers we have to change the template, and I think we have two options here:
1. Go back to the journal-style template, which would give an uglier look to the list of authors.
2. Not include running headers in this template.

I am hesitant to create our own edition of the IEEEtran template. If IEEE ever changes its template and we would like to follow it, who will remember that there's some code about stripping headers(lines 4315:4319) that must be removed?

**Comment by mrtz-j** [comment]
I tend to agree with option 2. here, but would it be possible to adjust it inside the latex file instead of the cls?

**Comment by nohr-ai** [comment]
Perhaps, I'm waiting to see what changes Mike is doing for the OS course template

**Comment by mrtz-j** [comment]
Any updates on this?
